docs/api_call.md

```diff
@@ -1,1352 +1,1154 @@
-        ]
-      }
-      ```
+  - *Django Shell:* 
 
-  - Example 2:
+    ```python
+    >>> message='i want to order chinese from mainland china and pizza from dominos'
+    >>> entity_name='restaurant' # here detection NER will search for values in dictionary 'restaurant'  
+    >>> structured_value=None
+    >>> fallback_value=None
+    >>> bot_message=None
+    >>> source_language='en'
+    
+    >>> from ner_v1.chatbot.entity_detection import get_text
+    >>> output = get_text(message=message, entity_name=entity_name,
+                          structured_value=structured_value,
+                          fallback_value=fallback_value,
+                          bot_message=bot_message,language=source_language)
+    >>> print(output)
+    
+    ```
 
-    - ```python
-      message = ''
-      entity_name = 'person_name'
-      structured_value = None
-      fallback_value = 'sagar nimesh dedhia'
-      bot_message = 'what is your name ?'
-      ```
+    The above can also be done from within the Docker container's shell. Setup is in docker.md file.
 
-    - *Python:*
+  - *CURL command:*
 
-      ```python
-      from ner_v1.chatbot.entity_detection import get_person_name
-      output = get_person_name(message=message, entity_name=entity_name, structured_value=structured_value, fallback_value=fallback_value, bot_message=bot_message)
-      print output
-      ```
+    ```shell
+    URL='localhost'
+    PORT=8081
+    
+    curl -i 'http://'$URL':'$PORT'/v1/text/?message=i%20want%20to%20order%20chinese%20from%20%20mainland%20china%20and%20pizza%20from%20domminos&entity_name=restaurant&structured_value=&fallback_value=None&bot_message=&source_language=en'
+    
+    ```
 
-    - *CURL command:*
+    > **Output **:
 
-      ```python
-      URL='localhost'
-      PORT=8081
-      ```
+    ```json
+    {"data": [
+        {
+            "detection": "message",
+            "original_text": "mainland china",
+            "entity_value": {"value": "Mainland China"},
+            "language": "en"
+        },
+        {
+            "detection": "message",
+            "original_text": "dominos",
+            "entity_value": { "value": "Domino's Pizza"},
+            "language": "en"
+        }]
+    }
+    
+    ```
+
+
+
+- **Example 2: *Detecting text entity from structured value***
+
+  - *Django Shell:* 
+
+    ```python
+    >>> message = 'मेरे लिए कैब बुक कर दीजिये'
+    >>> entity_name = 'city'
+    >>> structured_value = 'मुंबई'
+    >>> fallback_value = None
+    >>> bot_message = None
+    >>> source_langauge='hi'
+    
+    >>> from ner_v1.chatbot.entity_detection import get_text
+    >>> output = get_text(message=message, entity_name=entity_name,
+                        structured_value=structured_value,
+                        fallback_value=fallback_value,
+                        bot_message=bot_message,langauge=source_language)
+    >>> print(output)
+    
+    ```
+
+  - *CURL command:*
+
+    ```shell
+    URL='localho
```

Comment:
~This functionality helps to~ Detect entities that match by the specified pattern. If you are not familiar with regex, please see http://www.rexegg.com/regex-quickstart.html